ajax/libs/fomantic-ui/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "fomantic-ui",
+  "filename": "semantic.min.js",
+  "description": "Community fork of the popular Semantic-UI framework",
+  "keywords": [
+    "semantic",
+    "fomantic",
+    "framework",
+    "ui"
+  ],
+  "author": "hammy2899",
+  "homepage": "https://fomantic-ui.com",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fomantic/Fomantic-UI.git"
+  },
+  "autoupdate": {
```

Comment:
Would you like to use npm auto-update instead? I saw there are v2.3.1/2.3.2 on it but not on GitHub.

Reply:
Already discussed https://github.com/cdnjs/cdnjs/pull/12942#issuecomment-418687983

Reply:
Sorry for the missing